stubs/replay-tools.c

```diff
@@ -7,13 +7,13 @@ bool replay_events_enabled(void)
     return false;
 }
 
-int64_t replay_save_clock(unsigned int kind, int64_t clock, int64_t raw_icount)
+int64_t replay_save_clock(ReplayClockKind kind, int64_t clock, int64_t raw_icount)
```

Comment:
I see you imported the change from QEMU mainstream. Please cherry-pick the whole commit of theirs, it will make rebase easier.